packages/project-build/src/db/build.ts

```diff
@@ -216,17 +216,18 @@ const createPages = async (
     createNewTreeData({ projectId, buildId }),
     client
   );
-  return Pages.parse({
```

Comment:
The parse is here because the zod type has additional validation like this https://github.com/webstudio-is/webstudio-builder/blob/f6150c98da78331473254d213533f86f5e64d5ff/packages/project-build/src/schema/pages.ts#L70-L74

Same below. 

Please don't delete `parse` arbitrarily at least when you suspect that I added it. I usually try to add only when needed.

Reply:
Got it, thanks!

Reply:
Though do we need to validate data written in code? Kinda same as with props.

Reply:
I think we should when conditions are:

- Schema contains additional validations not just types (otherwise TS can do the same)
- No big performance concerns like large data or in hot path

We really want to avoid writing invalid data to DB